lib/matplotlib/tests/test_backend_pdf.py

```diff
@@ -1,9 +1,5 @@
 # -*- encoding: utf-8 -*-
```

Comment:
can drop the encoding cookie as well, py3 defaults to utf8 (https://docs.python.org/3/tutorial/interpreter.html#source-code-encoding)